modules/configuring-pipelines-as-code-for-git-repository-hosting-service-provider.adoc

```diff
@@ -0,0 +1,22 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: REFERENCE
+[id="configuring-pipelines-as-code-for-git-repository-hosting-service-provider_{context}"]
+= Configuring Pipelines as Code for Git repository hosting service provider 
+
+[role="_abstract"]
+After installing Pipelines as Code, cluster administrators can configure the Git repository hosting service provider. Currently, the following services are supported:
+
+* Github App
+* Github Webhook
+* Gitlab
```

Comment:
```suggestion
```
it will be out in pac 0.6.x

Reply:
in other word, not for Openshift Pipelines 1.7 and should not be mentioned